{"uuid": "6452cbde-817b-4eb9-8f4e-e568df698af3", "children": ["4a9ffbed-842a-4645-872a-6a10df131b18"], "befores": [{"name": "xunit_setup_class_fixture_TestLogout", "status": "passed", "start": 1634368300315, "stop": 1634368300315}], "afters": [{"name": "xunit_setup_class_fixture_TestLogout::0", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'TestLogout' has no attribute 'driver'\n", "trace": "  File \"d:\\python37\\lib\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"d:\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 941, in _teardown_yield_fixture\n    next(it)\n  File \"d:\\python37\\lib\\site-packages\\_pytest\\python.py\", line 815, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"d:\\python37\\lib\\site-packages\\_pytest\\python.py\", line 739, in _call_with_optional_argument\n    func(arg)\n  File \"E:\\Test_case\\Python\\app_test\\case\\test_logout.py\", line 19, in teardown_class\n    self.driver.close_app()\n"}, "start": 1634368309694, "stop": 1634368312703}], "start": 1634368300315, "stop": 1634368312703}